### ISSUE TITLE
Registered Trolltag

### ISSUE THREAD

**Issue by flanbun** [illuminatedwax/pesterchum#274]
I claimed the trolltag grimAuxiliatrix last night and at the time it was working just fine. However, after I reopened the application this morning it's saying that it's taken? Just very confused.

**Comment by pedanticPalindrome** [comment]
I know this is old and you probably figured it out by now, but I figured that other people will encounter a similar problem. 
On pesterchum, you may have noticed that there are no passwords (at least most of the time, there are a few exceptions). This is because anybody can use any username, so things like official character's handles dont get hogged by any one person (esp when people dont use them). What happened in this scenario is that it was currently in use. Character handles are only allowed to be used for 3 hours at a time (as far as I know) and obviously by only one person at a time. Many people like to use the handles for roleplay purposes, so this way its more fair. 
A common solution to those who want the same handle as an existing character is keeping the same acronym and spirit of the handle, without it being exactly the same. ie: gardenGnostical, trenchantTestification, timepieceGodless (some i found active just now)
If you want a username all to yourself, theres a way to register a password, but what it really does is kick off anyone using different wifi with that username (i think even if they do have the password). youd have to reset the command everytime you plan to switch wifi for a long time (such as moving), and its not practical for mobile thinks like laptops.  
TLDR: it was taken. anyone can use it, but only one at a time. pick a unique handle ^w^
